public/css/analytics.css

```diff
@@ -0,0 +1,24 @@
+.u-legend.u-inline .u-value {
+    width: 150px;
+    text-align: left;
+}
+
+#uplot {
+    background-color: #BBB;
```

Comment:
Perhaps use a css variable for the colors, easier to maintain and allows for theming (dark/light/...)